kudo-nginx-operator/manifests/preflight.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: troubleshoot.replicated.com/v1beta1
```

Comment:
same for preflight, we should consider removing the defaults that never pass and are more for demonstration that anything else